CRM/Utils/Money.php

```diff
@@ -368,6 +373,14 @@ public static function missingIntlNotice() {
    * @return int
    */
   protected static function getDecimalPlacesForAmount(string $amount): int {
+    if (!str_contains($amount, '.')) {
+      return 0;
+    }
+
+    if (is_string($amount) && str_contains($amount, '.')) {
```

Comment:
@braders @eileenmcnaughton just flagging this seems to assume that the $amount is in the US format (which it might be by this stage)